lib/msf/core/post/windows/shadowcopy.rb

```diff
@@ -182,6 +182,25 @@ def start_vss
         return false
       end
     end
+    unless start_swprv
+      return false
+    end
+    return true
+  end
+
+  def start_swprv
+    vss_state = wmic_query('Service where(name="swprv") get state')
```

Comment:
furthermore, service_info pulls way more information than what we need here. wmic allows us to pull only the one piece of specific data we care about. AFAICT this is a strictly superior approach here.